misc/images/materialized/entrypoint.sh

```diff
@@ -88,13 +88,21 @@ if [ -n "${MZ_RESTART_ON_FAILURE:-}" ]; then
     for ((i = 0; i < ${MZ_RESTART_LIMIT:-9999999999}; i++)); do
         # Run `environmentd` inside of an `if` to avoid tripping `set -e`
         # behavior.
+        # WIP: What's up with these identical branches? Should I maybe roll my
```

Comment:
It has to be an `if` so that we don't change the result code, otherwise we could `environmentd || true` to prevent `set -e` from killing the script after failure.